### DISCUSSION TITLE
how to disable spawn protection on tshock?

### DISCUSSION THREAD

**Discussion by alaexor** [Pryaxis/TShock#2965] [General]
I just wanted to play with my friends
but we are prevented by spawn protection because we cannot improve the arena and make houses

**Comment by sgkoishi** [comment]
[config-file-descriptions](https://github.com/Pryaxis/TShock/blob/general-devel/docs/config-file-descriptions.md) 😉 
![image](https://github.com/Pryaxis/TShock/assets/9637711/17fdc55a-4f19-448b-89dc-3a9251d4583c)

**Reply by ghost** [reply]
is this from bing chat @sgkoishi
